### PR TITLE
ABACUS Delete Button

### PR DESCRIPTION
**Pull request by kpals**
PLAT 1841 ticket: ABACUS delete functionality

### CODE REVIEW

File: src/containers/AttributeListItem/AttributeListItem.tsx [hunk 1]
```diff
@@ -130,6 +130,70 @@ const AttributeListItem: FC<Props> = (props) => {
...
+  const handleDeleteClick = useCallback(async () => {
+    const data = {
+      authority: activeAuthority,
+      name: activeAttribute?.name,
+      order: activeOrderList,
+      rule: activeAttribute?.rule,
+      state: activeAttribute?.state,
+    };
+
+    /* try {
```
Comment (pflynn-virtru) [inline src/containers/AttributeListItem/AttributeListItem.tsx:142]: delete this comment

File: src/containers/AttributeListItem/AttributeListItem.tsx [hunk 2]
```diff
@@ -130,6 +130,70 @@ const AttributeListItem: FC<Props> = (props) => {
...
+      toast.error("Could not delete attribute!");
+    } */
+
+    try {
+      await attributesClient.delete('/definitions/attributes', {
+        data,
+      });
+      //await getAuthorities();
+      toast.success(`Authority ${activeAttribute?.name} deleted`);
+      /* if (activeAuthority === authority) {
```
Comment (pflynn-virtru) [inline src/containers/AttributeListItem/AttributeListItem.tsx:159]: delete this comment

**Review by pflynn-virtru — DISMISSED** [review]

File: src/components/OrderCard/OrderCard.tsx [hunk 3]
```diff
@@ -39,7 +41,7 @@ const OrderCard: FC<Props> = (props) => {
 
   const actions = useMemo(() => {
     const config = {
-      view: [{ onClick: toggleEdit, key: "edit", text: "Edit Rule" }],
+      view: [{ onClick: toggleEdit, key: "edit", text: "Edit Rule" },{ onClick: onDeleteAttribute, key: "edit", text: "Delete Rule" }],
```
Comment (elizabethhealy) [inline src/components/OrderCard/OrderCard.tsx:44]: nit: maybe update the text to "Delete Attribute" as instead of "Delete Rule"

Comment (elizabethhealy) [reply src/components/OrderCard/OrderCard.tsx:44]: more of a design decision though

Comment (pflynn-virtru) [reply src/components/OrderCard/OrderCard.tsx:44]: "Delete Attribute" I second this

**Review by pflynn-virtru — DISMISSED** [review]

**Review by pflynn-virtru — APPROVED** [review]

**Pull request merged**